homeassistant/components/sma/sensor.py

```diff
@@ -84,7 +84,7 @@ def __init__(
     @property
     def name(self) -> str:
         """Return the name of the sensor."""
-        return self._sensor.name
+        return f"{DOMAIN}_{self._sensor.name}"
```

Comment:
This should be a user-friendly representation?

Why not set it to `<name of device info> <sensor name>`?

Reply:
Changed this with c7ded58